### ISSUE TITLE
Default should be en-GB

### ISSUE THREAD

**Issue by getsnoopy** [thundernest/thunderbird-website#359]
It seems like the website has multiple issues regarding defaults and language detection:

### Issue 1
**Problem**: If the browser's locale is not in the `xx-YY` format but is still BCP 47 compliant, it doesn't work properly. For example, my browser's locale is set to `en-GB-oxendict`, but the website chooses `en-US` anyway.

**Solution**: Resolve BCP 47 tags correctly and fall back to the most appropriate locale.

### Issue 2
**Problem**: It seems like [the website maps `en` to `en-US`](https://github.com/thundernest/thunderbird-website/blob/master/settings.py#L32), which is essentially [the incorrect choice for 74% of the English-speaking world](https://en.wikipedia.org/wiki/List_of_countries_by_English-speaking_population) that is not the US, Liberia, or the Philippines.

**Solution**: It should be `en-GB` instead, which is what the overwhelming majority of the world uses.

### Issue 3
**Problem**: On the home page, the language chooser in the footer has the label **English** and **English (British)**, etc. for the other Englishes, giving the false sense—when the drop-down is closed—that there are no other English choices, and—when the drop-down is open—that US English is the "default English of the world", which is the opposite of the truth.

**Solution**: Either **English** should map to `en-GB`, or the label for US English should change to **English (American)** to be consistent and clear about what the user is getting.

**Comment by Sancus** [comment]
1. Yeah, we could probably improve this, https://pypi.org/project/python-bcp47/ would likely do a better job than the current code.
2. The default is the base content language on this repository, content is written in en-US first and then translated to other languages. This is the same mapping mozilla.org used. `en` is a fallback code, no app should ever use it as the primary. Chrome, Firefox, and Edge all specify region for English. So this isn't really an issue in practice.
3. I agree, it should be `English (United States)`. That can be easily changed [here](https://github.com/thundernest/thunderbird-website/blob/master/website/_includes/lang_switcher.html#L20).

**Comment by gabldotink** [comment]
I agree “English” and “English (British),” “English (Canadian),” etc. is not ideal, and “English (American)” would be preferable. (A somewhat better system of labels might be “English (United States),” etc., but that’s a different discussion.)

I disagree with your assertion that `en-GB` should be the default, since the site is originally written in American English (setting aside the very dubious claim that “the overwhelming majority of the world” speaks British English).

**Comment by getsnoopy** [comment]
@gabldotink It's not dubious at all; American English is only use by ~26% of the English-speaking world, while the rest of the world learns Commonwealth English (of which British English is one variant), which the map I linked to shows. I'm not sure why this even needs explaining.

As for the default, while it might've been written in American English, I'm saying that's not a good reason to continue that practice. It doesn't serve the overwhelming majority of the English-speaking population of the world.
